src/test/java/reposense/parser/RepoConfigurationTest.java

```diff
@@ -260,4 +262,39 @@ public void repoConfig_sameLocationDifferentBranch_notEqual() throws InvalidLoca
 
         Assert.assertNotEquals(validLocationDefaultBranchRepoConfig, validLocationValidBranchRepoConfig);
     }
+
+    @Test
+    public void repoConfig_overrideStandaloneConfig_success()
+            throws ParseException, GitCloneException, IOException, HelpScreenException {
+        RepoConfiguration expectedConfig = new RepoConfiguration(new RepoLocation(TEST_REPO_DELTA), "master");
+
+        List<Author> expectedAuthorList = new ArrayList<>();
+        Author[] authors = new Author[]{FIRST_AUTHOR, SECOND_AUTHOR, THIRD_AUTHOR, FOURTH_AUTHOR};
+        for (Author author : authors) {
+            Author expectedAuthor = new Author(author);
+            List<String> expectedAuthorIgnoreGlobList = new ArrayList<>();
+            expectedAuthorIgnoreGlobList.addAll(author.getIgnoreGlobList());
+            expectedAuthorIgnoreGlobList.removeAll(REPO_LEVEL_GLOB_LIST);
```

Comment:
Add a comment here, explaning that the `Author`s original `ignoreGlobList` contains values from standalone config repo level, thus needs to be removed here